src/components/FormConfigurator.tsx

```diff
@@ -106,10 +107,12 @@ const FormConfigurator = ({
   const { query, replace } = useRouter();
 
   const [isImported, setIsImported] = useState(false);
+  const [configureRefetchIntervalForTokenAccounts, setConfigureRefetchIntervalForTokenAccounts] = useState(
```

Comment:
actually we dont need this right ? just use the form value ? and set it to undefined if we toggle it off

Reply:
good point. i have removed this `useState` and use the one in form value